python/agents/camel/camel/camel_library/interpreter/camel_value.py

```
@@ -0,0 +1,2003 @@
+    def get(self, key: _KV) -> _VV:
+        dict_key = next((el for el in self.iterate_python() if el.eq(key)), None)
+        if dict_key is None:
+            raise KeyError(key)
+        return self.python_value[dict_key].new_with_dependencies((self, key))
+
+    def eq(self, value: "Value") -> "CaMeLBool":
+        if not isinstance(value, type(self)):
+            return CaMeLFalse(camel_capabilities.Capabilities.camel(), (self, value))
+        for (self_k, self_v), (value_k, value_v) in zip(
+            self.python_value.items(), value.python_value.items()
+        ):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)